source/common/upstream/cluster_manager_impl.cc

```diff
@@ -1045,8 +1045,10 @@ void ClusterManagerImpl::updateClusterCounts() {
 }
 
 ThreadLocalCluster* ClusterManagerImpl::getThreadLocalCluster(absl::string_view cluster) {
+  if (!tls_.get()) {
+    return nullptr;
+  }
```

Comment:
during upstream filter initialization, the filter tries to do a warmup and perform a request. this object is not initialized at that moment. Is there a better solution for this?